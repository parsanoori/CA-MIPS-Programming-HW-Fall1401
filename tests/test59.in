99
866312576
653086568
255944315
513665235
440108178
580450291
70598635
8594972
957354576
747502623
1056277952
344628449
739419673
534022358
454331038
173064405
379389444
279611179
1024357684
1070324107
703998413
529729618
1048427774
17015433
312869706
544866603
1035443484
633722545
182536347
636187134
953381467
541536710
137210203
596099493
834324305
193787411
442844847
190891651
322017213
576502389
157968086
224902509
194172359
349310272
354947021
997286353
579908393
40689993
321382768
198940434
1031667735
259818107
845648692
749748912
400246832
637495651
485456497
660618458
473190857
236180605
965593467
355975188
203084892
379765460
255246913
637767026
681980742
687322109
581866182
216066783
871518641
989177769
611843760
257546962
519730328
991512197
285889697
143166376
753441466
1025531856
984644906
39008588
188147815
1058332455
1014871686
399858593
118974090
266536873
267072109
488761252
591189744
561706646
194497580
866897411
1032193829
740796300
449637933
652914635
500007913