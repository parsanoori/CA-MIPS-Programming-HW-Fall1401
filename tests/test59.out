8594972
17015433
39008588
40689993
70598635
118974090
137210203
143166376
157968086
173064405
182536347
188147815
190891651
193787411
194172359
194497580
198940434
203084892
216066783
224902509
236180605
255246913
255944315
257546962
259818107
266536873
267072109
279611179
285889697
312869706
321382768
322017213
344628449
349310272
354947021
355975188
379389444
379765460
399858593
400246832
440108178
442844847
449637933
454331038
473190857
485456497
488761252
500007913
513665235
519730328
529729618
534022358
541536710
544866603
561706646
576502389
579908393
580450291
581866182
591189744
596099493
611843760
633722545
636187134
637495651
637767026
652914635
653086568
660618458
681980742
687322109
703998413
739419673
740796300
747502623
749748912
753441466
834324305
845648692
866312576
866897411
871518641
953381467
957354576
965593467
984644906
989177769
991512197
997286353
1014871686
1024357684
1025531856
1031667735
1032193829
1035443484
1048427774
1056277952
1058332455
1070324107